App/Screens/Home/Home.tsx

```diff
@@ -63,6 +63,32 @@ function getCigarettesMargin(count: number): number {
 	);
 }
 
+/*
+ * Dynamically calculating max number of cigarettes
+ */
+function getDynamicMaxCigarettes(count: number): number {
+	const CIGARETTE_ASPECT_RATIO = 21 / 280; // taken from the @shootismoke/ui lib
+	const height = scale(
+		count <= THRESHOLD.FIRST
+			? SIZES.BIG
+			: count <= THRESHOLD.SECOND
+			? SIZES.MEDIUM
+			: count <= THRESHOLD.THIRD
+			? SIZES.BIG
+			: count <= THRESHOLD.FOURTH
+			? SIZES.MEDIUM
+			: SIZES.SMALL
+	);
```

Comment:
This code is used twice, could you factor it out in a separate function `getCigarettesHeight`?

Reply:
Changes applied :+1: